packages/components/molecules/f-searchbox/stories/Searchbox.stories.js

```diff
@@ -1,10 +1,11 @@
-// Uncomment the import below to add prop controls to your Story (and add `withKnobs` to the decorators array)
-import {
-    select, boolean
-} from '@storybook/addon-knobs';
 import Vue from 'vue';
 import Vuex from 'vuex';
 import { withA11y } from '@storybook/addon-a11y';
+import {
+    ENGLISH_LOCALE,
+    SPANISH_LOCALE
+} from '@justeat/storybook/constants/globalisation';
+
```

Comment:
Is it possible to add the globalisation for the other tenants too? So IE, IT, AU, NZ, DK & NO as well

Reply:
Of course. I just used the ones there were constants for, so wasn't sure what else there was. But I've found a small list in the `cookie-banner` stories.

Reply:
Amazing, thank you!